lib/ex_ami/message.ex

```diff
@@ -144,8 +144,12 @@ defmodule ExAmi.Message do
   def is_event(%Message{} = message), do: is_type(message, "Event")
 
   def is_response_success(%Message{} = message) do
```

Comment:
Shouldn't this be named:
```suggestion
  def response_success?(%Message{} = message) do
```
And similarly for the other functions above and below?

Reply:
It should by today's standards. However,  this is old code and beyond the scope of this PR.